switchbot/__init__.py

```diff
@@ -51,6 +62,19 @@ def _disconnect(self) -> None:
         finally:
             self._device = None
 
+    def _commandkey(self, key) -> str:
+        key = ""
```

Comment:
??

Reply:
Yes that is strange.

I'll look into it and get back